packages/snap-preact-components/src/components/Atoms/Terms/Terms.test.tsx

```diff
@@ -0,0 +1,221 @@
+import { h } from 'preact';
+
+import { v4 as uuidv4 } from 'uuid';
+import { render } from '@testing-library/preact';
+import { ThemeProvider } from '../../../providers';
+
+import { Terms } from './Terms';
+import { AutocompleteController, AutocompleteControllerConfig } from '@searchspring/snap-controller';
+import { createAutocompleteController } from '@searchspring/snap-preact';
+import { MockClient } from '@searchspring/snap-shared';
+import userEvent from '@testing-library/user-event';
+
+describe('Terms Component', () => {
+	const globals = { siteId: '8uyt2m' };
+	let controllerConfigId = uuidv4().split('-').join('');
+	let acConfig: AutocompleteControllerConfig = {
+		id: controllerConfigId,
+		selector: 'input.searchspring-ac',
+		settings: {
+			trending: {
+				limit: 5,
+			},
+			history: {
+				limit: 5,
+			},
+		},
+	};
+	const clientConfig = {
+		globals: {
+			siteId: '8uyt2m',
+		},
+	};
+
+	const mockClient = new MockClient(globals, {});
+	mockClient.mockData.updateConfig({ meta: 'ac.meta' });
+
+	let mockTerms = [
+		{
+			active: false,
+			preview: () => {},
+			value: 'dress',
+			url: {
+				href: 'www.dress.com',
+			},
+		},
+		{
+			active: false,
+			preview: () => {},
+			value: 'drss',
+			url: {
+				href: 'www.drss.com',
+			},
+		},
+		{
+			active: false,
+			preview: () => {},
+			value: 'dres',
+			url: {
+				href: 'www.dres.com',
+			},
+		},
+		{
+			active: false,
+			preview: () => {},
+			value: 'dss',
+			url: {
+				href: 'www.dss.com',
+			},
+		},
+		{
+			active: false,
+			preview: () => {},
+			value: 'ress',
+			url: {
+				href: 'www.ress.com',
+			},
+		},
+	];
+
+	let controller: AutocompleteController;
+	beforeEach(async () => {
+		controller = createAutocompleteController({ client: clientConfig, controller: acConfig }, { client: mockClient });
+		await controller.bind();
+	});
+
+	it('renders', async () => {
+		// @ts-ignore - typing on terms
```

Comment:
I'd like to see these ignores go away if possible. I know we've had typing issues in our tests for a long time. Was this due to the `'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.ts(2686)` error?

It may be worth digging into it and trying to resolve fully for this and all the other tests where this is happening.